src/universal-app/kitchen-sink/kitchen-sink.ts

```diff
@@ -55,7 +54,7 @@ export class KitchenSink {
 
   /** Data source for the CDK and Material table. */
   tableDataSource: DataSource<any> = {
-    connect: () => Observable.of([
+    connect: () => of([
```

Comment:
Before, when Material2 imports the rxjs "_functions_" directly they used to rename the `of` import like this:

`import {of as observableOf} from 'rxjs/observable/of';`

Not sure if it's correct to use just `of...`.

Reply:
Probably because `of` is a keyword in the ES6 `for...of` syntax. I'll change it just to be safe.

Reply:
Yes. In fact, I thought `rxjs` could change this operator name, as they did for (to avoid conflicts):

`catch` -> `catchError`
`do` -> `tap`
`finally` -> `finalize`
`switch` -> `switchAll`

Reply:
It might be worth filing an issue in the [rxjs repo](https://github.com/ReactiveX/rxjs)